addons/ness-lambda/@{{ cookiecutter.root }}/@{{ cookiecutter.lambda_name }}.tf

```diff
@@ -0,0 +1,181 @@
+locals {
+  # Additional policies to attach to the lambda
+  lambda_@{{ cookiecutter.lambda_underscore }}_policies = [
+    {% if cookiecutter.is_triggered_by_sqs == "true" -%}
+    "arn:aws:iam::aws:policy/service-role/AWSLambdaSQSQueueExecutionRole", # Needed to allow Lambda manage sqs, see https://docs.aws.amazon.com/lambda/latest/dg/services-sqs-configure.html#events-sqs-permissions
+    {% endif -%}
+  ]
+}
+
+variable "@{{ cookiecutter.lambda_underscore }}" {
+  description = "Configuration of the @{{ cookiecutter.lambda_name }} workflow"
+  type = object({
+    {% if cookiecutter.is_triggered_by_sqs == "true" -%}
+    sqs_batch_size                        = number
+    {% endif -%}
+    cloudwatch_logs_retention_in_days     = number
+    lambda_reserved_concurrent_executions = number
+    lambda_timeout                        = number
```

Comment:
Should we also add `studio_secret` ?

Reply:
> Should we also add `studio_secret` ?

I choose not to put it here because it will be a global variable for all lambdas

Reply:
What about a comment, so we (or PAM) do not forget about it? We define the var at the line 47.